Packs/CrowdStrikeFalconX/Integrations/CrowdStrikeFalconX/CrowdStrikeFalconX.py

```diff
@@ -0,0 +1,511 @@
+    :return: a dict based on api_current_dict without the keys that doesn't appear in fields_to_keep
+    """
+    if not api_current_dict or not fields_to_keep:
+        return {}
+
+    group_outputs = {}
+
+    for field_to_keep in fields_to_keep:
+        if field_to_keep in api_current_dict.keys():
+            group_outputs[field_to_keep] = api_current_dict.get(field_to_keep)
+
+    return group_outputs
+
+
+def parse_outputs(
+        api_res: Dict,
+        meta_fields: list = [],
+        quota_fields: list = [],
+        resources_fields: list = [],
+        sandbox_filds: list = []
+) -> Dict[str, dict]:
+    """Parse group data as received from CrowdStrike FalconX API into Demisto's conventions
+    the output from the API is a dict that contains the keys: meta, resources and errors
+    the meta contains a "quota" dict
+    the "resources" is an array that contains the sandbox dict
+    the function filters the wanted params from the api result
+    :param api_res: the api result from the http request
+    :param meta_fields: the wanted params that appear in the mate section
+    :param quota_fields: the wanted params that appear in the quota section
+    :param resources_fields: the wanted params that appear in the resources section
+    :param sandbox_filds: the wanted params that appear in the sandbox section
+    :return: a dict based on api_res with the wanted params only
+    """
+    if api_res.get("errors"):
+        # if there is an error in the api result, return only the error
+        return api_res.get("errors")
+
+    api_res_meta, api_res_quota, api_res_resources, api_res_sandbox = {}, {}, {}, {}
+    resources_group_outputs, sandbox_group_outputs = {}, {}
+
+    api_res_meta = api_res.get("meta")
+    if api_res_meta:
+        api_res_quota = api_res_meta.get("quota")
+
+    meta_group_outputs = add_outputs_from_dict(api_res_meta, meta_fields)
+    quota_group_outputs = add_outputs_from_dict(api_res_quota, quota_fields)
+
+    if api_res.get("resources"):
+        # depended on the command, the resources section can be a str list or a list that contains
+        # only one argument which is a dict
+        if type(api_res.get("resources")[0]) == dict:
+            api_res_resources = api_res.get("resources")[0]
+            resources_group_outputs = add_outputs_from_dict(api_res_resources, resources_fields)
+
+            if api_res_resources and api_res_resources.get("sandbox"):
+                api_res_sandbox = api_res_resources.get("sandbox")[0]
+                sandbox_group_outputs = add_outputs_from_dict(api_res_sandbox, sandbox_filds)
+        else:
+            # the resources section is a list of strings
+            resources_group_outputs = {"resources": api_res.get("resources")}
+
+    merged_dicts = {**meta_group_outputs, **quota_group_outputs, **resources_group_outputs, **sandbox_group_outputs}
+
+    return {f'csfalconx.resource(val.resource === obj.resource)': merged_dicts}
```

Comment:
I'm not sure that this is how the merging DT works. please try it. IMO it's something like:
```suggestion
    return {f'csfalconx.resource(val.id === obj.id)': merged_dicts}
```

Reply:
the merging was tested and works
the context has been changed